ci/jenkins/jobs/macros.yaml

```diff
@@ -231,7 +231,10 @@
           # Clean up dangling images generated in previous builds. Recent ones must be excluded
           # because they might be being used in other builds running simultaneously.
           docker image prune -f --filter "until=1h" || true
-          VERSION="$JOB_NAME-$BUILD_NUMBER" make
+          for i in `seq 2`
+          do
+            VERSION="$JOB_NAME-$BUILD_NUMBER" make && break
```

Comment:
What will happen if by accident the second retry fails? For example there is some other problem in the system other than the 1 hour 'docker prune'.

Reply:
Then it fails and testbed remains. I think it is a rare case and the testbed will be cleaned by GC.

Reply:
I tried, it's like:

```
#!/bin/bash
set -e

for i in `seq 2`; do false && break; done

echo It continues to run even failed
```
```
./test-bash.sh
It continues to run even failed
```

Then

```
#!/bin/bash
set -e

for i in `seq 2`; do false && break; done

if [ "$?" -ne "0" ]; then
  echo Failed
  exit 1
fi

echo It continues to run even failed
```
```
./test-bash.sh
Failed
```

I think this is because, if cmd1 in  `cmd1 && ...` fails, even with `set -e`, bash will ignore the error. So we should detect the error after the loop.

Reply:
I misunderstood before. Yes you are right. Updated.